daily-scrum/planning.yaml

```diff
@@ -10,51 +10,36 @@ messages:
   # Friday week0
   0:
```

Comment:
We also need to change the starting date, since offsets have been removed, or else the message for monday will be sent friday midnight, won't it?

Reply:
Thank you, fixed